lib/fog/compute/kubevirt/models/servers.rb

```diff
@@ -0,0 +1,32 @@
+require 'fog/core/collection'
+require 'fog/ovirt/models/compute/server'
+
+module Fog
+  module Compute
+    class Kubevirt
+      class Servers < Fog::Collection
+        attr_reader :kind, :resource_version
+
+        model Fog::Compute::Kubevirt::Server
+
+        def all(filters = {})
+          servers = service.list_servers(filters)
+          @kind = servers.kind
+          @resource_version = servers.resource_version
+          load servers
+        end
+
+        def get(id)
+          new service.get_server(id)
+        end
+
+        def bootstrap(new_attributes = {})
```

Comment:
Please document potential `new_attributes`

Reply:
will do that in a next PR's version